### PR TITLE
Add: Image  & input

### PR DESCRIPTION
**Pull request by saifudinbadani**
Screen Shot 2022-02-07 at 3 06 01 pm
Screen Shot 2022-02-07 at 3 05 56 pm
Screen Shot 2022-02-07 at 3 05 34 pm

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "pwa-chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (nikhil-sharma-nks) [inline .vscode/launch.json:15]: if this file is not necessary then you can add it to gitignore

File: components/card/card.css [hunk 2]
```diff
@@ -1 +1,89 @@
...
+.text-overlay{
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    border-radius: 7px;
+    height: 100%;
+    width: 100%;
+    background-color: #fffdffc9;
```
Comment (nikhil-sharma-nks) [inline components/card/card.css:83]: i think you have a typo in hex-color

File: components/card/card.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+                    <small class="card-author">by Studium</small>
+                    <p class="card-text heading-4">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet, unde!
+                    </p>
+                    <div class="card-footer">
+                        <button class="card-btn btn btn-solid-primary">Add to Cart</button>
+                    </div>
+                </div>
+            </div> -->
+
+
```
Comment (nikhil-sharma-nks) [inline components/card/card.html:150]: if this is not needed then you should remove this comment

**Review by nikhil-sharma-nks — COMMENTED** [review]
if not necessary then consider removing 
- .vscode file
- comments
- extra empty line
Other than that looks good to me.

File: .vscode/launch.json (hunk 1 above)
Comment (SHIKHASINGH1506) [reply .vscode/launch.json:15]: avoid pushing .vs files

**Review by SHIKHASINGH1506 — COMMENTED** [review]
Added a comment, rest are good to go

File: .vscode/launch.json (hunk 1 above)
Comment (saifudinbadani) [reply .vscode/launch.json:15]: Done that.

File: components/card/card.css (hunk 2 above)
Comment (saifudinbadani) [reply components/card/card.css:83]: It is because of transparency.

File: components/card/card.html (hunk 3 above)
Comment (saifudinbadani) [reply components/card/card.html:150]: I am making this one as card with badge

**Pull request merged**